### ISSUE TITLE
Updates to UC10

### ISSUE THREAD

**Issue by marisalim** [nih-cfde/use-case-library-build#53]
Add things to edit in the use case (e.g., description, tasks, reqs) - https://use-cases.nih-cfde.org/uc-00010/

**Comment by josesanchez1815** [comment]
Haven't been able to work on this particular use case with Selenium yet, but I can leave comments at the end of today.

**Comment by josesanchez1815** [comment]
t-0028: Notify team data administrator there are Review Catalogs to review
  - I didn't see a button or function for this when I was going through the website. How is this action normally done? which user has permissions to notify the data administrator? Or is the data administrator notified when there is actual metadata submissions entered? @ACharbonneau @marisalim @abhijna @s-canchi

**Comment by s-canchi** [comment]
I think I made a note of the same point and considered it to be a perhaps future functionality that is yet to be implemented.

**Comment by marisalim** [comment]
yes, this point also applies to uc9 - #50. i've added an in progress emoji next to task 28 for now. when #56 is merged, it will show up across the UCL.

as we do more testing (i.e., more coming end of March), the emoji can change to the [approved and/or done emoji](https://use-cases.nih-cfde.org/CONTRIBUTING/#use-case-approval-process) :)
